spec/azure/storage.go

```diff
@@ -0,0 +1,5 @@
+package azure
+
+type Storage struct {
+	AccountType string `json:"accountType" yaml:"accountType"`
```

Comment:
Lets have the comments you provided with the PR description also here in the code. This is more sustainable and helpful for everybody coming by.

Reply:
Done :)